### ISSUE TITLE
Change Quota Size

### ISSUE THREAD

**Issue by colemanja** [davidschlachter/outlookquota#7]
Hi, love this addin, been looking for something like this for a long time.
Is there a way to reduce the Quota size down to 1GB.

Jamie

**Comment by davidschlachter** [comment]
Yes — you'd just change the quota size on [line 15 of QuotaTool.vb](https://github.com/davidschlachter/outlookquota/blob/5f302f7eca53d9fb947457a3af6e4efaf9ee2d01/EmailSizer/QuotaTool.vb#L15) to 1000000000 instead of 2000000000 bytes. Then just compile the project and distribute as you wish.

I'm not developing on Windows right now so I wouldn't be able to make this change myself. When I had made the add-on I saw it being centrally deployed, so the quota [wasn't made to be user-configurable](https://github.com/davidschlachter/outlookquota/issues/2) (but easy to change when building the add-on, as above).
